src/lib/components/code-block.svelte

```diff
@@ -40,7 +40,7 @@
 </script>
 
 {#if content || content === null}
-  <div class="relative w-full h-full">
+  <div class="relative w-full h-auto">
```

Comment:
This will probably cause a merge conflict with some changes I just made. But, I think I fixed all of the issues in #453.